cost-analyzer/templates/awsstore-deployment-template.yaml

```diff
@@ -29,6 +29,10 @@ spec:
       {{- if .Values.awsstore.priorityClassName }}
       priorityClassName: "{{ .Values.awsstore.priorityClassName }}"
       {{- end }}
+      {{- with .Values.nodeSelector }}
```

Comment:
I'm thinking we should give the option to use a specific node selector for this. 
`with .Values.awsstore.nodeSelector`
No?

Reply:
I have no strong opinion on this, either is fine to me

Reply:
I agree with Jesse here. Because this is an esoteric component, I would rather pull this value from `awsstore.nodeSelector` than from the top-level key `nodeSelector`.